client/app/domain/translations/Messages_fr_FR.xml

```diff
@@ -146,6 +146,7 @@
    <translation id="domain_tab_DNS_unlock_warning_manuel_check_tool" qtlid="293749">Outil de vérification DNS</translation>
 
    <translation id="domain_tab_DNSSEC_action_add_title" qtlid="37875">Modifier</translation>
+   <translation id="domain_tab_DNSSEC_action_delete_entry">Supprimer</translation>
```

Comment:
You can apply `qtlid="2414"` or on the view side you could use `common_delete` as translations key.

Reply:
Fixed 😄